website/source/docs/configuration/interpolation.html.md

```diff
@@ -80,6 +80,31 @@ The supported built-in functions are:
   * `base64encode(string)` - Returns a base64-encoded representation of the
     given string.
 
+  * `cidrfirstaddr(iprange)` - Takes an IP address range in CIDR notation
+    and returns the first IP address in that range. This is the same
+    as simply removing the prefix indicator from the end. For example,
+    ``cidrfirstaddr("10.0.0.0/8", 2)`` returns ``10.0.0.0``.
+
+  * `cidrhost(iprange, hostnum)` - Takes an IP address range in CIDR notation
+    and creates an IP address with the given host number. For example,
+    ``cidrhost("10.0.0.0/8", 2)`` returns ``10.0.0.2``.
+
+  * `cidrlastaddr(iprange)` - Takes an IP address range in CIDR notation
+    and returns the last IP address in that range. For example,
+    ``cidrlastaddr("10.0.0.0/8", 2)`` returns ``10.255.255.255``.
```

Comment:
Similar question here. I wonder if we should shift and expand these functions to match up with the `ipcalc` naming scheme that distinguishes between network/broadcast and hostmin/hostmax?